INSTALL.md

```diff
@@ -0,0 +1,124 @@
+
+To install cocoapods, we need the Xcode command line tools and brew. In a terminal enter:
+
+    xcode-select --install
+
+To install brew, follow the instructions at https://brew.sh . To install cocoapods, in a terminal enter:
+
+    brew install cocoapods
+
+## Ubuntu 18.04, 20.04 and 22.04
+
+To install Git, in a terminal, enter:
+
+    sudo apt install git
+
+To install go 1.17, download go<version>.linux-amd64.tar.gz from https://go.dev/dl.
+In a terminal, enter for following with the correct <version> to install in /usr/local .
+
+    sudo tar -C /usr/local -xzf go<version>.linux-amd64.tar.gz
+
+To install Python module pystache, in a terminal enter:
+
+    sudo apt install python3-pip
+    sudo python3 -m pip install pystache
+
+(Python module yaml is already installed.)
+
+To install Android Studio, download the latest android-studio-<version>-linux.tar.gz from https://developer.android.com/studio .
+(Tested with Chipmunk 2021.2.1 .) In a terminal, enter the following with the correct <version>:
+
+    sudo tar -C /usr/local -xzf android-studio-<version>-linux.tar.gz
+
+To launch Android Studio, in a terminal enter:
+
+    /usr/local/android-studio/bin/studio.sh &
+
+Build for Android
+=================
+* Launch Android Studio and accept the default startup options. Create a new default project (so that we get the main screen).
+* On the Tools menu, open the SDK Manager.
+* In the "SDK Platforms" tab, check "Android 10.0 (Q)".
+* In the "SDK Tools" tab, click "Show Package Details". Expand "NDK (Side by side)" and check "23.1.7779620".
+* Click OK to install and close the SDK Manager.
+* If you are not plugging in a real phone, on the Tools menu open the Device Manager and create an Android 10 device.
+
+Open a new terminal to get the setup from the installers. To set the environment variables, enter the following,
+depending on your platform.
+(You must do this each time you start a new terminal, or put the commands in ~/.bash_profile.)
+
+macOS:
+
+    export GOPATH="$HOME/go"
+    export PATH="$PATH:$GOPATH/bin"
+    export ANDROID_HOME="$HOME/Library/Android/sdk"
+    export ANDROID_NDK_HOME="$ANDROID_HOME/ndk/23.1.7779620"
+    export PATH="$PATH:$ANDROID_HOME/emulator"
+    export PATH="$PATH:$ANDROID_HOME/platform-tools"
+    export JAVA_HOME="/Applications/Android Studio.app/Contents/jre/Contents/Home"
+
+Ubuntu:
+
+    export GOPATH="$HOME/go"
+    export PATH="$PATH:$GOPATH/bin"
+    export PATH="$PATH:/usr/local/go/bin"
+    export ANDROID_HOME="$HOME/Android/Sdk"
+    export ANDROID_NDK_HOME="$ANDROID_HOME/ndk/23.1.7779620"
+    export PATH="$PATH:$ANDROID_HOME/emulator"
+    export PATH="$PATH:$ANDROID_HOME/platform-tools"
+    export JAVA_HOME="/usr/local/android-studio/jre"
+    export PATH="$PATH:$JAVA_HOME/bin"
+
+In a terminal, enter:
+
+    git clone https://github.com/ipfs-shipyard/gomobile-ipfs
+    cd gomobile-ipfs/packages
+    make build_core.android
+    make test_bridge.android
```

Comment:
It seems to me that the rule `make test_bridge.android` will launch `make build_core.android` by itself.

I think you could also explain to people that this part tells them how to run the tests on the bridge; someone might only want to build the core or the Example app and would not be interested in this step at all.

Reply:
You are right. I will update it.

Reply:
I changed it to say look at `make help` and that other targets build the core.